src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -207,14 +208,11 @@ export default class AirportModel {
         // airspace perimeter (assumed to be first entry in data.airspace)
         this.perimeter = _head(this.airspace);
 
-        // change ctr_radius to point along perimeter that's farthest from rr_center
-        // const pos = new PositionModel(this.perimeter.poly[0].position, this.position, this.magnetic_north);
-
         this.ctr_radius = Math.max(..._map(
-            this.perimeter.poly, (v) => vlen(
+            this.perimeter.poly, (vertexPosition) => vlen(
                 vsub(
-                    v.position,
-                    PositionModel.calculatePosition(this.rr_center, this.position, this.magnetic_north)
+                    vertexPosition.relativePosition,
```

Comment:
oh thank goodness!  I can now associate a word with this, I always thought `v` was for `value` 

👍 👍